CursedMod/Features/Wrappers/Player/RagDolls/CursedRagDoll.cs

```diff
@@ -1,4 +1,10 @@
-﻿using Mirror;
+﻿using CursedMod.Features.Wrappers.Server;
+using InventorySystem.Items.Pickups;
+using Mirror;
+using PlayerRoles;
+using PlayerStatsSystem;
+using System;
+using System.CodeDom;
```

Comment:
System.CodeDom?

Reply:
I don't know where but I am

Reply:
well, you can always delete the reference if you're not using it